### PR TITLE
chore(daemon): removing password, entropy and validator number as prompt, and adding flags instead

### PR DESCRIPTION
**Pull request by kehiy**
## Description

In this PR I removed the validator number, password, and entropy as prompt and hard-coded values. now we get them as flags from the user, this helps other interfaces and clients like GUI use this `init` command with ease.

### CODE REVIEW

File: cmd/daemon/init.go [hunk 1]
```diff
@@ -53,15 +60,12 @@ func buildInitCmd(parentCmd *cobra.Command) {
...
-		cmd.PrintLine()
-		cmd.PrintInfoMsgf("Enter a password for wallet")
-		password := cmd.PromptPassword("Password", true)
 
-		cmd.PrintLine()
-		cmd.PrintInfoMsgBoldf("How many validators do you want to create?")
-		cmd.PrintInfoMsgf("Each node can run up to 32 validators, and each validator can hold up to 1000 staked coins.")
-		cmd.PrintInfoMsgf("You can define validators based on the amount of coins you want to stake.")
-		numValidators := cmd.PromptInputWithRange("Number of Validators", 7, 1, 32)
+		if *valNumOpt < 1 || *valNumOpt > 32 {
```
Comment (b00f) [inline cmd/daemon/init.go:64]: We can't change the previous behavior.
I suggest:
1- Don't set default in flags
2- If it is not set (or zero) ask user to set it manually

**Review by b00f — CHANGES_REQUESTED** [review]

File: cmd/daemon/init.go [hunk 2]
```diff
@@ -53,15 +59,12 @@ func buildInitCmd(parentCmd *cobra.Command) {
 			err := wallet.CheckMnemonic(*restoreOpt)
 			cmd.FatalErrorCheck(err)
 		}
-		cmd.PrintLine()
-		cmd.PrintInfoMsgf("Enter a password for wallet")
-		password := cmd.PromptPassword("Password", true)
 
-		cmd.PrintLine()
```
Comment (b00f) [inline cmd/daemon/init.go:60]: Still we break the previous behaviour. If user didn't set the number of validator, it should work as before.

File: cmd/daemon/init.go [hunk 3]
```diff
@@ -53,15 +59,12 @@ func buildInitCmd(parentCmd *cobra.Command) {
...
-		cmd.PrintInfoMsgBoldf("How many validators do you want to create?")
-		cmd.PrintInfoMsgf("Each node can run up to 32 validators, and each validator can hold up to 1000 staked coins.")
-		cmd.PrintInfoMsgf("You can define validators based on the amount of coins you want to stake.")
-		numValidators := cmd.PromptInputWithRange("Number of Validators", 7, 1, 32)
```
Comment (b00f) [inline cmd/daemon/init.go:64]: Still we break the previous behaviour. If user didn't set the number of validator, it should work as before.

Comment (b00f) [reply cmd/daemon/init.go:64]: It should be like:
```
var valNum int
if *valNumOpt == 0 {
// .... old code
} else {
// ... new code
}
```

**Comment by kehiy** [comment]
@b00f fixed.

**Review by b00f — APPROVED** [review]

**Pull request merged**